packages/@okta/vuepress-site/docs/guides/sampleapp-oie-redirectauth/main/index.md

```diff
@@ -101,11 +101,13 @@ This section walks you through enabling self-service enrollment for the Sign-In
 
 This section walks you through the self-service enrollment steps for a new user.
```

Comment:
Generally speaking, these changes detail using Okta email with OTP. The description of the related PR (EML: Update Redirect auth use cases article to add magic link coverage) discusses the magic link flow where the user clicks on a link in the email to complete the verification. Am I reading this right?

Reply:
You're absolutely right. Fixing.